docs/encyclopedia/temporal-nexus-service-operations.mdx

```diff
@@ -0,0 +1,166 @@
+---
+id: nexus-services-operations
+title: Nexus Services and Operations
+sidebar_label: Nexus Service and Operations
+description: Discover how Nexus Service and Operations enable the creation of well-defined API contracts for sharing capabilities across team and Namespace boundaries, leveraging Temporal's durable and scalable infrastructure.
+toc_max_heading_level: 4
+slug: /temporalnexus/nexus-services-operations
+keywords:
+  - Nexus Services
+  - Nexus Operations
+  - Operation Lifecycle
+  - durable execution
+  - Automatic Retries
+  - Execution Semantics
+  - Temporal SDK
+  - Versioning
+tags:
+  - nexus 
+  - temporal nexus 
+  - nexus services
+  - nexus operations
+
+---
+
+The following page covers the following: 
+- [Nexus Services](#nexus-services)
+- [Nexus Operations](#nexus-services) 
+   - [Operation Lifecycle](#operation-lifecycle)
+   - [Automatic Retries](#automatic-retries)
+   - [Execution Semantics](#execution-semantics)
+   - [Versioning](#versioning)
+
+## Nexus Services {#nexus-services}
+
+A Nexus Service is a named collection of arbitrary-duration Nexus Operations that provide an API contract suitable for sharing across team boundaries. 
+Nexus Services are registered with a Temporal Worker that is listening on the target Namespace and Task Queue for an Endpoint.
+
+For example, a Nexus Service is often registered in the same Worker as the underlying Workflows they abstract:
+
+```go
+func main() {
+	c, err := client.Dial(client.Options{})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	w := worker.New(c, taskQueue, worker.Options{})
+	service := nexus.NewService(service.HelloServiceName)
+	err = service.Register(handler.EchoOperation, handler.HelloOperation)
+	if err != nil {
+		log.Fatalln("Unable to register operations", err)
+	}
+	w.RegisterNexusService(service)
+	w.RegisterWorkflow(handler.HelloHandlerWorkflow)
+
+	err = w.Run(worker.InterruptCh())
+	if err != nil {
+		log.Fatalln("Unable to start worker", err)
+	}
+}
+```
+
+## Operations {#nexus-operations}
+
+Nexus Operations are not intrinsically durable, but can be made durable by backing them with Temporal primitives like a Workflow, Update, or Signal. 
+The Temporal SDK provides helper functions to create Operations that may be registered with a Service in a Temporal Worker.
+
+Calling a Nexus Operation from a caller Workflow presents as a non-blocking, arbitrary duration operation. This schedules the Operation with the caller’s Temporal Service and does not block caller Workflow execution. 
+The caller’s Nexus Machinery is responsible for making the underlying Nexus RPC calls and updating the caller’s Workflow history with the result.
+
+### Operation Lifecycle {#operation-lifecycle}
+
+The Temporal Go SDK is used for the operation lifecycle examples below.
```

Comment:
Would convert the Go SDK specific things to a generic, SDK independent description